examples/ABMC.yml

```diff
@@ -0,0 +1,14 @@
+abbreviation: ABMC
+departments:
+- name: American Battle Monuments Commission
+  submission_methods:
+  - submission_format: usps
+    name: Beatriz Collazo
+    title: FOIA Assistant
```

Comment:
I was thinking:
```yaml
    address_lines:
    - Beatriz Collazo
    - FOIA Assistant
    - Suite 500
    - 2300 Clarendon Boulevard
    - Arlington, VA 22201
```

Although now realizing the [2015 contact data](https://github.com/18F/2015-foia/blob/master/contacts/data/ABMC.yaml#L4) parses out city/state/zip:

```yaml
address:
    address_lines:
    - Martha R. Sell
    - FOIA Assistant
    - Suite 500
    city: Arlington
    state: VA
    street: 2300 Clarendon Boulevard
    zip: '22201'
```